__tests__/app.test.js

```diff
@@ -180,4 +180,39 @@ describe("GET /api/articles/:article_id/comments", ()=> {
             expect(msg).toBe("Bad Request")
         })
      });
-});
\ No newline at end of file
+});
+
+describe("POST /api/articles/:article_id/comments", ()=> {
+    test("201: posts a comment for the article id that was input", () => {
+        const testComment = {
+            username: "icellusedkars",
+            body: "Great Website"
+        };
+        return request(app)
+        .post("/api/articles/9/comments")
+        .send(testComment)
+        .expect(201)
+        .then((response) => {
+            const { body: { comment }} = response
+            expect(comment).toEqual(
+                expect.objectContaining({
+                    body: "Great Website",
+                    votes: 0,
+                    author: "icellusedkars",
+                    article_id: 9,
+                    created_at: expect.any(String),
+                    comment_id: expect.any(Number),
+                })
+            )
+        });
+    });
+     test("400: returns the correct error message when invalid id is input", ()=> {
+        return request(app)
+        .post("/api/articles/not-a-number/comments")
+        .send(({ username: "icellusedkars", body: "Great Website"}))
+        .expect(400)
+        .then(({ body: {msg} }) => {
+            expect(msg).toBe("Bad Request")
+        })
+     });
```

Comment:
There's quite a few more tests you can add here! 

- /api/articles/99999/comments 
- what if the body sent over has content missing? 
- what if the username isn't one that's in the database? 
- what if the incorrect data type is used for body (a number instead of a string?)